ang/test/civicase/case/actions/directives/case-actions.directive.spec.js

```diff
@@ -140,6 +140,34 @@
           expect(element.isolateScope().isActionEnabled(action)).toBe(false);
         });
       });
+
+      describe('when using write actions', () => {
```

Comment:
Lets use something like `when using actions that makes changes to the case`, because `write action can be confusing`